identity/pom.xml

```diff
@@ -122,10 +122,6 @@
           <groupId>com.google.guava</groupId>
           <artifactId>guava-jdk5</artifactId>
         </exclusion>
-        <exclusion>
-          <groupId>com.google.api-client</groupId>
```

Comment:
just for understanding, why was this change needed?

Reply:
Making this change was allowing both `mvn clean install` and the integration tests to work correctly. I do not have ac concrete explanation for now. This is a bit hacky.